bin/review-rot

```diff
@@ -98,11 +98,33 @@ def main(cli_args, valid_choices):
                     )
                 )
 
-    # Now, with all results in place, sort them and print
+    # With the --sort argument, --comment-sort is kept for backwards
+    # compatibility. Equivalent to --sort commented
     if arguments.get('comment_sort'):
+        arguments['sort'] = 'commented'
+
+    # Now, with all results in place, sort them and print
+    # If sort is not passed or not provided as configuration argument, use the
+    # first element of the sort choices list as default.
+    sort_by = arguments.get('sort', choices['sort'][0])
+
+    if sort_by == 'commented':
         sorting_key = sort_by_last_comment
     else:
-        sorting_key = operator.attrgetter('time')
+        sort_index = {
+            'submitted': 'time',
+            'updated': 'updated_time',
+        }
+        sort_attr = sort_index.get(sort_by)
+
+        # As sort_index has to be explicitly match, ensure that is kept in sync
+        # with sort choices list.
+        if sort_attr is None:
+            error_message = 'Sort by {} not supported'.format(sort_by)
+            log.debug(error_message)
```

Comment:
What's the purpose of a debug log message here? Since it's an error you are logging, this should probably use `log.error`.

Reply:
I was trying to be consistent with the rest of the code. See lines 54 to 56:

```
        if 'type' not in item:
            log.debug('git service type not found for %s', item)
            raise KeyError('git service not found for %s' % item)
```